crates/astria-sequencer/src/accounts/query.rs

```diff
@@ -88,7 +88,7 @@ pub(crate) async fn balance_request(
         Err(err_rsp) => return err_rsp,
```

Comment:
At the top of this function (i.e. `fn balance_request`), please add a doc comment that explains that it will return the balances sorted by denom and amount.